advanced/server-extension/README.md

```diff
@@ -345,11 +406,12 @@ by calling the `finish` method. That method can optionally take an argument that
 become the response body of the request in the frontend.
 
 ```py
-# server_extension/handlers.py#L14-L16
+# jlab_ext_example/handlers.py#L23-L26
```

Comment:
Is this snippet off by a few lines?

Reply:
yes, it is correct, as the beginning of the file has changed also.